src/region/RegionAnnotations.scss

```diff
@@ -18,7 +18,12 @@
 }
 
 .ba-RegionAnnotations-popup {
+    display: none;
```

Comment:
I think we'll want to leave the popup visible but maybe disable the textarea/button during submission, instead. Can you check with design?